devices/Ssd13xx/IFont.cs

```diff
@@ -1,4 +1,8 @@
-﻿using System;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
 
 namespace Iot.Device.Ssd13xx
```

Comment:
Note not for this PR: we should rellay have a look at all the IFint using CharacterLcd or extracting something. Will make life easier to many people!